pkgs/applications/misc/plater/default.nix

```diff
@@ -17,8 +18,23 @@ mkDerivation rec {
     sha256 = "0r20mbzd16zv1aiadjqdy7z6sp09rr6lgfxhvir4ll3cpakkynr4";
   };
 
-  nativeBuildInputs = [ cmake ];
-  buildInputs = [ libGLU qtbase ];
+  nativeBuildInputs = [ cmake qt5.wrapQtAppsHook ];
```

Comment:
I am not fully sure here but I think not passing in qt5 is preferred because then versions are not mixed and set by ``libsForQt5.callPackage``.

Reply:
Yeah, I don't know how this is supposed to look, sadly :/

Reply:
@SuperSandro2000 Figured it out by reading the manual; fixed!